util/test/sub_test

```diff
@@ -1041,11 +1041,14 @@ for testname in testsrc:
                 if suppressme:
                     with open('./'+test_filename+'.suppressif', 'r') as suppressfile:
                         suppressline = suppressfile.readline().strip()
-                    if suppressline.startswith("#"):
-                      suppressline = suppressline.replace("#","");
-                    else:
-                      suppressline = ""
-                    suppressline = suppressline.strip()
+                        if suppressline.startswith("#!"):
```

Comment:
I might change this to 

```python 

                    # grab first non-shebang comment and use as a short description
                    suppressline = ''
                    with open('./'+test_filename+'.suppressif', 'r') as suppressfile:
                        for line in suppressfile:
                            line = line.strip()
                            if not line.startswith('#!') and line.startswith('#'):
                                suppressline = line.replace('#').strip()
```

This way the comment doesn't have to be on the first/second line to be picked up.